etc/ci/performance/test_differ.py

```diff
@@ -49,4 +53,13 @@ def load_data(filename):
         diff = value2 - value1
         change = diff / value1
         color = BLUE if value1 <= value2 else GREEN
+        total1 += value1
+        total2 += value2
         print("{}{:6} {:6} {:+6} {:+8.2%}   {}.{}".format(color, value1, value2, diff, change, key, END))
+
+
+print()
```

Comment:
nit: If the intent is to print a empty line, I would explicitly put an empty string between the parens.

Reply:
Fixed.